docs/src/components/DocLayout/primitives/StyledMain.tsx

```diff
@@ -1,11 +1,13 @@
 import styled from "styled-components";
 
+import { MAX_CONTENT_WIDTH } from "../../../consts";
+
 const StyledMain = styled.main`
   flex: 1;
   position: relative;
   display: flex;
-  padding-top: 6px; /* align with DocNavigation */
   flex-direction: column;
+  max-width: calc(${MAX_CONTENT_WIDTH} - 22ch);
```

Comment:
Why 22ch? Could this be some meaningful named const?

Reply:
because the navigation bar has width `22ch`, decided not to change that. Initially i was more for using `rems` everywhere, just fyi ;)